js-exercises/negative-index/negativeIndex.js

```diff
@@ -0,0 +1,35 @@
+function isNumber(numberValue) {
+  if (!numberValue || typeof numberValue === "symbol") {
+    return false;
+  }
+  numberValue = Number(numberValue);
+  return typeof numberValue === "number" && !isNaN(numberValue);
+}
+
+function negativeIndex(array) {
+  if (!Array.isArray(array)) {
+    throw TypeError("Only arrays are supported.");
+  }
+
+  const handlers = {
+    get(target, property, receiver) {
+      let getThisIndex = property;
+      if (isNumber(property) && property < 0) {
+        getThisIndex = parseInt(getThisIndex);
+        getThisIndex = target.length + getThisIndex;
+      }
+      return Reflect.get(target, getThisIndex, receiver);
+    },
+    set(target, property, value) {
+      let setThisIndex = property;
```

Comment:
Same is the case with `setThisIndex`. We want to set/modify a value at that particular `index`.
Not a big change but from a readability perspective, it will be good if you rename it!